temporian/core/operators/since_last.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample operator class and public API function definition."""
+
+from typing import Optional
+
+from temporian.core import operator_lib
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.operators.base import Operator
+from temporian.proto import core_pb2 as pb
+from temporian.core.data.dtype import DType
+
+
+class SinceLast(Operator):
+    def __init__(
+        self,
+        event: Event,
+        sampling: Optional[Event] = None,
+    ):
+        super().__init__()
+
+        self.add_input("event", event)

```

Comment:
Remember to change these too when merging main (renamed standard "event" name with "input" and "output" respectively)